postgres/literal.go

```diff
@@ -11,6 +11,11 @@ func Bool(value bool) BoolExpression {
 	return CAST(jet.Bool(value)).AS_BOOL()
 }
 
+// BoolArray creates new bool array literal expression
+func BoolArray(elements []bool) BoolArrayExpression {
+	return jet.BoolArray(elements)
```

Comment:
We'll probably need an explicit cast here as well, since cockroachdb tends to have problem with untyped parameters.